Arduino/startUpwithBluetooth/startUpwithBluetooth.ino

```diff
@@ -0,0 +1,70 @@
+#include <Smartcar.h>
+
+const unsigned short LEFT_ODOMETER_PIN = 2;
+const unsigned short RIGHT_ODOMETER_PIN = 3;
+const unsigned long PRINT_INTERVAL = 100;
+unsigned long previousPrintout = 0;
+const int TRIGGER_PIN = 6; //D6
+const int ECHO_PIN = 7; //D7
+const unsigned int MAX_DISTANCE = 100;
+char blue;
+SR04 front(TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);                //bluetooth value reciever
+
+
+BrushedMotor leftMotor(8, 10, 9);
+BrushedMotor rightMotor(12, 13, 11);
+DifferentialControl control(leftMotor, rightMotor);
+
+GY50 gyroscope(37);
+DirectionlessOdometer leftOdometer(100);
+DirectionlessOdometer rightOdometer(100);
```

Comment:
`100` is not the correct pulses-per-meter ratio. Please find the appropriate value for your sensor.

Reply:
recalculated the value for pulse per meter for the odometers which was between 180-190 so that will be updated too.